shacl12-overview/index.html

```diff
@@ -346,30 +346,118 @@
+			</p>
+			<p>
+				The <em>What's new in SHACL 1.2</em> section below describes the changes between SHACL, as published in 2017 and SHACL 1.2, including all the new SHACL elements such as SHACL for User Interface generation.
+			</p>
+			<p>
+				The <em>Specifications</em> section below lists all the Data Shapes Working Group's specifications which are also listed in each specification's Introduction.
+			</p>
+			<p>
+				The appendixes below contain material supporting the set of SHACL specifications and provide SHACL users with additional support, such as validators for shapes graphs.
+			</p>
 		</section>
 
-		<section id="shapes">
-			<h2>Shapes</h2>
-			<p class="todo">TODO: This section will describe the Shapes created for SHACL-SHACL.</p>
+		<section id="whatsnew">
+			<h2>What's New in SHACL 1.2</h2>
 			<p>
-				These shapes are available in the RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
+				<em>coming soon</em>
 			</p>
 		</section>
 
-		<section id="governance">
-			<h2>Governance</h2>
-			<p class="todo">TODO: This section will describe the governance arrangements in place for the continued maintenance of this NOTE.</p>
+		<section id="specifications">
+			<h2>Specifications</h2>
+			<p>
+				SHACL 1.2:
+			</p>
+			<ol>
+				<li>SHACL 1.2 Overview (this document)
+					<ul><li>overviews the set of SHACL specifications</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-core/">SHACL 1.2 Core</a>
+					<ul><li>defines the Core of SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-sparql/">SHACL 1.2 SPARQL Extensions</a>
+					<ul><li>defines SPARQL-related extensions of the SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-node-expr/">SHACL 1.2 Node Expressions</a>
+					<ul><li>defines graph expressions used to determine focus nodes SHACL</li></ul>
+				</li>
+				<li>
+					<a href="https://w3c.github.io/data-shapes/shacl12-inf-rules/">SHACL 1.2 Inference Rules</a>
+					<ul><li>defines SHACL's methods of rule-based inference</li></ul>
+				</li>
+				<li>
+					<s><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 UI</a></s>
+					<ul><li>defines SHACL's use for User Interface generation</li></ul>
+				</li>
+				<li>
+					<s><a href="https://w3c.github.io/data-shapes/shacl12-profiling/">SHACL 1.2 Profiling</a></s>
+					<ul><li>defines the use of SHACL for profiling data, including SHACL data</li></ul>
+				</li>
+			</ol>
+		</section>
```

Comment:
I do think we need to address the governance question near the description of SHACL-SHACL.  This question alone might determine whether SHACL-SHACL needs its own NOTE.  The below suggestion restores the governance section, though shifted down to here.

```suggestion
			</ol>

			<section id="governance">
				<h3>Governance</h3>
				<p class="todo">TODO: This section will describe the governance arrangements in place for the continued maintenance of SHACL-SHACL resources, i.e., handling updates beyond the time frame of the Working Group's Charter.</p>
			</section>
		</section>
```

Reply:
Pardon: restores, with some extra English.

Reply:
No worries: I've added it back in